### PR TITLE
[fix] Async migration file was not ignored

### PR DESCRIPTION
**Pull request by darthsoup**
During our process of upgrading the Package to the next most recent version, we encountered the problem that we could not continue without the migration. But we don't need this migration.

While investigating the problem i saw that the ServiceProvider is using the old way `$this->publishes` and the new way (`$this->loadMigrationsFrom` - implemented since L5.3) at the same time. The `$this->publishes` call dont have the Check.
Due to the fact that the package only supports >= Laravel 5.5, we do not need the second call.

### CODE REVIEW

**Comment by lindyhopchris** [comment]
Ah the idea here is that if you want to customise the migration, you can publish them then edit them within your app, instead of the package loading the migrations.

As per these instructions:
https://github.com/cloudcreativity/laravel-json-api/blob/develop/docs/features/async.md#migration-customisation

So if you are using `LaravelJsonApi::runMigrations();` to load the migrations, then you should not also be publishing the migrations

TLDR use `runMigrations` or publish the migration - don't use both.

Does that fix you problem?

**Comment by darthsoup** [comment]
Argh crap, you're absolutely right. I completely forgot the part about custom migrations.
Reviewed my project again and found that it was deployment related that I couldn't do the migrations because we were running `vendor:publish --all`.

Thank for you help :)
